libs/embed/src/embed.ts

```diff
@@ -254,7 +259,7 @@ class Novu {
         );
       };
 
-      iframe.src = `${IFRAME_URL}/${clientId}`;
+      iframe.src = `${IFRAME_URL}/${clientId}/${backendUrl}`;
```

Comment:
I think best would be is to pass the extra params inside query params instead of URL params. For example: 
```suggestion
      iframe.src = `${IFRAME_URL}/${clientId}?`;
      if (backendUrl) {
          iframe.src = iframe.src + 'backendUrl=' + enchodeURICompnent(backendUrl);
      }
```

And then we need to modify the apps/widget code to extract the query params and pass them to the notification center NovuProvider implementation.

Reply:
I imagine that's more performant! For modifying the apps/widget code, specifically apps/widget/src/components/notification-center/NotificationCenterWidget.tsx, are these the kinds of modifications you're looking for?

(check the line before the } )
`interface INotificationCenterWidgetProps {
  onUrlChange: (url: string) => void;
  onNotificationClick: (notification: IMessage) => void;
  onUnseenCountChanged: (unseenCount: number) => void;
  applicationIdentifier: string | undefined;
  backendUrl: string;
}`


(check the line right under NovuProvider)
`<NovuProvider
        backendUrl={props.backendUrl as string}
        socketUrl={WS_URL}
        applicationIdentifier={props.applicationIdentifier as string}
        subscriberId={userDataPayload.subscriberId}
        onLoad={onLoad}
        subscriberHash={userDataPayload.subscriberHash}
      >`

Reply:
Now instead of just looking at these changes here you can checkout my new commit